### ISSUE TITLE
[Bug] Ore Dictionary registration for "slate" fails

### ISSUE THREAD

**Issue by CREEATION** [the-realest-stu/Rustic#246]
```
[23:39:44] [main/WARN]: ****************************************
[23:39:44] [main/WARN]: * Invalid registration attempt for an Ore Dictionary item with name slate has occurred. The registration has been denied to prevent crashes. The mod responsible for the registration needs to correct this.
[23:39:44] [main/WARN]: *  at net.minecraftforge.oredict.OreDictionary.registerOreImpl(OreDictionary.java:660)
[23:39:44] [main/WARN]: *  at net.minecraftforge.oredict.OreDictionary.registerOre(OreDictionary.java:646)
[23:39:44] [main/WARN]: *  at rustic.common.crafting.Recipes.addOreDictEntries(Recipes.java:149)
[23:39:44] [main/WARN]: *  at rustic.common.crafting.Recipes.initOres(Recipes.java:45)
[23:39:44] [main/WARN]: *  at rustic.core.CommonProxy.preInit(CommonProxy.java:50)
[23:39:44] [main/WARN]: *  at rustic.core.ClientProxy.preInit(ClientProxy.java:44)...
[23:39:44] [main/WARN]: ****************************************
[23:39:44] [main/WARN]: ****************************************
[23:39:44] [main/WARN]: * Invalid registration attempt for an Ore Dictionary item with name slate has occurred. The registration has been denied to prevent crashes. The mod responsible for the registration needs to correct this.
[23:39:44] [main/WARN]: *  at net.minecraftforge.oredict.OreDictionary.registerOreImpl(OreDictionary.java:660)
[23:39:44] [main/WARN]: *  at net.minecraftforge.oredict.OreDictionary.registerOre(OreDictionary.java:646)
[23:39:44] [main/WARN]: *  at rustic.common.crafting.Recipes.addOreDictEntries(Recipes.java:150)
[23:39:44] [main/WARN]: *  at rustic.common.crafting.Recipes.initOres(Recipes.java:45)
[23:39:44] [main/WARN]: *  at rustic.core.CommonProxy.preInit(CommonProxy.java:50)
[23:39:44] [main/WARN]: *  at rustic.core.ClientProxy.preInit(ClientProxy.java:44)...
[23:39:44] [main/WARN]: ****************************************
[23:39:44] [main/WARN]: ****************************************
[23:39:44] [main/WARN]: * Invalid registration attempt for an Ore Dictionary item with name slate has occurred. The registration has been denied to prevent crashes. The mod responsible for the registration needs to correct this.
[23:39:44] [main/WARN]: *  at net.minecraftforge.oredict.OreDictionary.registerOreImpl(OreDictionary.java:660)
[23:39:44] [main/WARN]: *  at net.minecraftforge.oredict.OreDictionary.registerOre(OreDictionary.java:646)
[23:39:44] [main/WARN]: *  at rustic.common.crafting.Recipes.addOreDictEntries(Recipes.java:151)
[23:39:44] [main/WARN]: *  at rustic.common.crafting.Recipes.initOres(Recipes.java:45)
[23:39:44] [main/WARN]: *  at rustic.core.CommonProxy.preInit(CommonProxy.java:50)
[23:39:44] [main/WARN]: *  at rustic.core.ClientProxy.preInit(ClientProxy.java:44)...
[23:39:44] [main/WARN]: ****************************************
[23:39:44] [main/WARN]: ****************************************
[23:39:44] [main/WARN]: * Invalid registration attempt for an Ore Dictionary item with name slate has occurred. The registration has been denied to prevent crashes. The mod responsible for the registration needs to correct this.
[23:39:44] [main/WARN]: *  at net.minecraftforge.oredict.OreDictionary.registerOreImpl(OreDictionary.java:660)
[23:39:44] [main/WARN]: *  at net.minecraftforge.oredict.OreDictionary.registerOre(OreDictionary.java:646)
[23:39:44] [main/WARN]: *  at rustic.common.crafting.Recipes.addOreDictEntries(Recipes.java:152)
[23:39:44] [main/WARN]: *  at rustic.common.crafting.Recipes.initOres(Recipes.java:45)
[23:39:44] [main/WARN]: *  at rustic.core.CommonProxy.preInit(CommonProxy.java:50)
[23:39:44] [main/WARN]: *  at rustic.core.ClientProxy.preInit(ClientProxy.java:44)...
[23:39:44] [main/WARN]: ****************************************
[23:39:44] [main/WARN]: ****************************************
[23:39:44] [main/WARN]: * Invalid registration attempt for an Ore Dictionary item with name slate has occurred. The registration has been denied to prevent crashes. The mod responsible for the registration needs to correct this.
[23:39:44] [main/WARN]: *  at net.minecraftforge.oredict.OreDictionary.registerOreImpl(OreDictionary.java:660)
[23:39:44] [main/WARN]: *  at net.minecraftforge.oredict.OreDictionary.registerOre(OreDictionary.java:646)
[23:39:44] [main/WARN]: *  at rustic.common.crafting.Recipes.addOreDictEntries(Recipes.java:153)
[23:39:44] [main/WARN]: *  at rustic.common.crafting.Recipes.initOres(Recipes.java:45)
[23:39:44] [main/WARN]: *  at rustic.core.CommonProxy.preInit(CommonProxy.java:50)
[23:39:44] [main/WARN]: *  at rustic.core.ClientProxy.preInit(ClientProxy.java:44)...
[23:39:44] [main/WARN]: ****************************************
[23:39:44] [main/WARN]: ****************************************
[23:39:44] [main/WARN]: * Invalid registration attempt for an Ore Dictionary item with name slate has occurred. The registration has been denied to prevent crashes. The mod responsible for the registration needs to correct this.
[23:39:44] [main/WARN]: *  at net.minecraftforge.oredict.OreDictionary.registerOreImpl(OreDictionary.java:660)
[23:39:44] [main/WARN]: *  at net.minecraftforge.oredict.OreDictionary.registerOre(OreDictionary.java:646)
[23:39:44] [main/WARN]: *  at rustic.common.crafting.Recipes.addOreDictEntries(Recipes.java:154)
[23:39:44] [main/WARN]: *  at rustic.common.crafting.Recipes.initOres(Recipes.java:45)
[23:39:44] [main/WARN]: *  at rustic.core.CommonProxy.preInit(CommonProxy.java:50)
[23:39:44] [main/WARN]: *  at rustic.core.ClientProxy.preInit(ClientProxy.java:44)...
[23:39:44] [main/WARN]: ****************************************
```

I have disabled slate worldgen in the config and also set `Slate veins per chunk` to `0`.

**Comment by katubug** [comment]
Having this same issue.

**Comment by the-realest-stu** [comment]
Fixed in version 1.1.1
